homeassistant/components/squeezebox/media_player.py

```diff
@@ -284,6 +290,23 @@ def supported_features(self):
         """Flag media player features that are supported."""
         return SUPPORT_SQUEEZEBOX
 
+    @property
+    def sync_group(self):
+        """List players we are synced with."""
+        player_ids = {p.unique_id: p.entity_id for p in self.hass.data[DATA_SQUEEZEBOX]}
+        sync_group = []
+        for player in self._player.sync_group:
+            if player in player_ids:
+                sync_group.append(player_ids[player])
+            else:
+                _LOGGER.debug("Squeezebox %s in sync_group but not an entity.", player)
```

Comment:
We don't want side effects like logging in entity properties.

If we want to tell users about the non entity players, maybe make a separate attribute for those?